README.md

```diff
@@ -361,6 +362,30 @@ the Ewon Flexy's web interface.
     - `http://{EWON-HOST-ADDRESS}/rcgi.bin/jvmForm?formName=statusApi&getVersion=true`
     - via M2Web: `https://m2web.talk2m.com/t2mapi/get/MyeWON/rcgi.bin/jvmForm?formName=statusApi&getVersion=true`
 
+#### Bootstrap/Provisioning Endpoints
+
+1. Set Bootstrap Authentication (setBootstrapAuth)
+    - Sets the bootstrap authentication configuration of the connector.
+    - `http://{EWON-HOST-ADDRESS}/rcgi.bin/jvmForm?formName=setBootstrapAuth&host={host}&port={port}&tenant={tenant}&username={username}&password={password}`
+    - via M2Web: `https://m2web.talk2m.com/t2mapi/get/MyeWON/rcgi.bin/jvmForm?formName=setBootstrapAuth&host={host}&port={port}&tenant={tenant}&username={username}&password={password}`
```

Comment:
```
    - via M2Web: `https://m2web.talk2m.com/t2mapi/get/{ewon-name}/rcgi.bin/jvmForm?formName=setBootstrapAuth&host={host}&port={port}&tenant={tenant}&username={username}&password={password}`
```
Aren't they removing support for username and pass and forcing session?

Reply:
Yes, they are changing the authentication mechanism. The parameters here are specific to setting the connector bootstrap configuration and do not include M2Web authentication. That is left up to the user to determine.